spec/services/search/instagram_service_spec.rb

```diff
@@ -1,3 +1,95 @@
 require 'rails_helper'
+require 'hashie'
 
-RSpec.describe Search::TwitterService, type: :service do; end
+RSpec.describe Search::InstagramService, type: :service do
+  subject { described_class }
+
+  context 'class methods' do
+    it { is_expected.to respond_to(:make_a_item_by) }
+    it { is_expected.to respond_to(:find_media_items_for) }
+  end
+
+  describe '.find_media_items_for' do
+    xit 'needs to return the crawled items from web' do
+      event = create :event, hash_tag: '#justin', services: [ServiceKind::INSTAGRAM]
+
+      expect(event.items.count).to eq(0)
+
+      subject.find_media_items_for(event)
+
+      expect(event.items.count).to eq(100)
+    end
+  end
+
+  describe '.make_a_item_by' do
+    let(:media_item) do
+      Hashie::Mash.new({ id: '596008529309278208',
+                         caption: { text: 'a caption text for image' },
+                         user: {
+                             id: '1831819079',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.